hashing/te-tag-query-java/com/facebook/threatexchange/Utils.java

```diff
@@ -158,13 +158,13 @@ public static void outputNonTMKHashToFile(
   {
     FileOutputStream fos = new FileOutputStream(path);
 
-    byte[] bytes = sharedHash.hashValue.getBytes();
+    byte[] bytes = sharedHash.td_raw_indicator.getBytes();
     fos.write(bytes);
     if (verbose) {
       SimpleJSONWriter w = new SimpleJSONWriter();
       w.add("path", path);
-      w.add("hash_type", sharedHash.hashType);
-      w.add("hash_length", sharedHash.hashValue.length());
+      w.add("hash_type", sharedHash.td_indicator_type);
+      w.add("hash_length", sharedHash.td_raw_indicator.length());
```

Comment:
should these also be changed from saying hash?

Reply:
good call